sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/postgres/PostgresNumericTests.java

```diff
@@ -547,6 +547,19 @@ public void testAdd() {
         this.testTwoViews(intermediate, last);
     }
 
+
+    @Test
+    public void testSafeAdd() {
```

Comment:
This file only contains tests from the postgres testsuite, as described in the first comment.
So I would move these tests to a separate file in sql, perhaps in the `simple` directory.
I would also write some new tests which overflow, that's the point of this function.
In fact, safe_add is a bigquery SQL extension, so you probably won't find tests for it in Postgres or mysql.